### DISCUSSION TITLE
Does the adjacent color clause of SC 1.4.11 apply when focus changes swap colors but also are adjacent to the background

### DISCUSSION THREAD

**Discussion by mraccess77** [w3c/wcag#1733] [Q&A]
In the screenshot below the button background and page background are the same color.  The focus indicator is communicated by a color change of the circular button to become darker.  SC 1.4.11 doesn't require the focused and unfocused states to have contrast - but must the new focused color contrast with adjacent page background colors as well because the focus indicator takes up the whole button area?

![image](https://user-images.githubusercontent.com/5640755/114189634-68b6b000-9918-11eb-9505-6b7432c39c17.png)

For a circular button - what is considered the component rather than page background - is it the bounding box or the visible component?

**Comment by mraccess77** [comment]
I have moved this to an issue as no one has responded in 3 weeks.

**Reply by patrickhlauke** [reply]
https://github.com/w3c/wcag/issues/1775

**Comment by bruce-usab** [comment]
I must confess, I never noticed this Discussions tab before!  It does seem like some things might be better suited as discussions as compared to issues.
